android/app/src/main/java/com/mattermost/share/ShareModule.java

```diff
@@ -78,6 +80,8 @@ public void close(ReadableMap data) {
                 }
             }
         }
+
+        RealPathUtil.deleteTempFiles(this.tempFolder);
```

Comment:
Is this `tempFolder` used for other purposes? If not, any reason we don't just delete all files? If yes, any reason not to create a folder dedicated as such and from which we can delete all files?

Reply:
Temp folder here is required cause by the time this is called we no longer have the context of the activity thus we cannot get the cache folder

Reply:
In the back of my mind I was thinking: can we dedicate this temp folder exclusively and just recursively delete the entire thing vs. picking out only select files for deletion?